src/compiler/scala/tools/cmd/CommandLineParser.scala

```diff
@@ -35,7 +35,15 @@ object CommandLineParser {
   }
   private object DoubleQuoted extends QuotedExtractor('"')
   private object SingleQuoted extends QuotedExtractor('\'')
-  private val Word = """(\S+)(.*)""".r
+  object Word {
+    private val regex = """(\S+)""".r
+    def unapply(s: String): Option[(String, String)] = {
+      regex.findPrefixOf(s) match {
+        case Some(prefix) => Some(prefix, s.substring(prefix.length, s.length()))
```

Comment:
I just meant a linter should save you time on this sort of style comment.

I hope you're catching some of the Mr Rogers marathon. He's so good.